pkg/gadgets/trace/mount/types/types.go

```diff
@@ -15,27 +15,57 @@
 package types
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64   `json:"mntnsid,omitempty"`
-	Pid       uint32   `json:"pid,omitempty"`
-	Tid       uint32   `json:"tid,omitempty"`
-	Comm      string   `json:"comm,omitempty"`
-	Operation string   `json:"operation,omitempty"`
-	Retval    int      `json:"ret,omitempty"`
-	Latency   uint64   `json:"latency,omitempty"`
-	Fs        string   `json:"fs,omitempty"`
-	Source    string   `json:"source,omitempty"`
-	Target    string   `json:"target,omitempty"`
-	Data      string   `json:"data,omitempty"`
-	Flags     []string `json:"flags,omitempty"`
+	Comm      string   `json:"comm,omitempty" column:"comm,maxWidth:16"`
+	Pid       uint32   `json:"pid,omitempty" column:"pid,minWidth:7"`
+	Tid       uint32   `json:"tid,omitempty" column:"tid,minWidth:7"`
+	MountNsID uint64   `json:"mntnsid,omitempty" column:"mnt_ns,width:12"`
+	Operation string   `json:"operation,omitempty" column:"op,minWidth:5,maxWidth:7,hide"`
+	Retval    int      `json:"ret,omitempty" column:"ret,width:3,fixed,hide"`
+	Latency   uint64   `json:"latency,omitempty" column:"latency,minWidth:3,hide"`
+	Fs        string   `json:"fs,omitempty" column:"fs,minWidth:3,maxWidth:8,hide"`
+	Source    string   `json:"source,omitempty" column:"src,width:16,hide"`
+	Target    string   `json:"target,omitempty" column:"dst,width:16,hide"`
+	Data      string   `json:"data,omitempty" column:"data,width:16,hide"`
+	Flags     []string `json:"flags,omitempty" column:"flags,width:24,hide"`
 	FlagsRaw  uint64   `json:"flagsRaw,omitempty"`
 }
 
+func GetColumns() *columns.Columns[Event] {
+	cols := columns.MustCreateColumns[Event]()
```

Comment:
I think it is missing configuring the extractor for flags:

https://github.com/kinvolk/inspektor-gadget/blob/5b93ab18b3b22741b26ae326d259156928d0cd54/cmd/kubectl-gadget/trace/mount.go#L149